src/tcgwars/logic/impl/gen4/SecretWonders.groovy

```diff
@@ -3522,10 +3522,10 @@ f
           text "You can play only one Supporter card each turn. When you play this card, put it next to your Active Pokémon. When your turn ends, discard this card.\nDraw 2 cards. Then, choose a card from your opponent’s hand without looking and put it on the bottom of his or her deck."
           onPlay {
             draw 2
-            opp.hand.select(hidden: true, "Choose a card from your opponent's hand without looking").showToOpponent("Team Galactic's Mars: This card will be put on the bottom of your deck").moveTo(hidden: true, opp.deck)
+            if (opp.hand) opp.hand.shuffledCopy().select(hidden: true, "Choose a card from your opponent's hand without looking").showToOpponent("Team Galactic's Mars: This card will be put on the bottom of your deck.").moveTo(hidden: true, opp.deck)
```

Comment:
We don't need shuffledCopy() anymore, that is handled by select itself now.

Reply:
Hmm, interesting. Will need to be removed from several cards at some point.

Reply:
Yeah, doesn't really hurt anything to have it, but no longer necessary, so best not to add more of them.